include/android-ashmem.h

```diff
@@ -105,7 +105,9 @@ static inline void *shmat(int __shmid, const void *__shmaddr, int __shmflg) {
 
 }
 
-#endif /* __ANDROID__ */
+#endif                                                       /* __ANDROID__ */
+
+#endif
```

Comment:
In total three `#if` and four `#endif`. That does not match! Kill the last `#endif`

Reply:
done